iceoryx_utils/test/moduletests/test_cxx_newtype.cpp

```diff
@@ -83,8 +83,20 @@ TEST(NewType, CopyAssignableDoesCompile)
 {
     Sut<cxx::NewType<int, newtype::ConstructByValueCopy, newtype::CopyAssignable, newtype::Comparable>> a(491), b(492),
         c(423);
+
+    // Ignore false positive
+    // See: https://isocpp.org/wiki/faq/strange-inheritance#hiding-rule
+
+#if (__GNUC__ == 9 && (__GNUC_MINOR__ > 1 || (__GNUC_MINOR__ == 1 && __GNUC_PATCHLEVEL__ > 0)))
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-copy"
```

Comment:
assuming we get the same warning when we use it elsewhere, we can't litter all our code with this. Okay for now, but we need to find a more viable solution